en/08_Changelogs/6.0.0.md

```diff
@@ -275,6 +276,24 @@ Sake used to have functionality to make daemon processes for your application. T
 
 We've removed this functionality. Please use an appropriate daemon tool such as `systemctl` to manage these instead.
 
+### Read-only replica database support {#db-read-only-replicas}
+
+Read-only replicas are additional databases that are used to offload read queries from the primary database, which can improve performance by reducing the load on the primary database.
+
+Read-only replicas are configured by adding environment variables that match the primary environment variable and suffixing `_REPLICA_<replica-number>` to the variable name, where `<replica_number>` is the replica number padding by a zero if it's less than 10, for example `SS_DATABASE_SERVER` becomes `SS_DATABASE_SERVER_REPLICA_01` for the first replica, or `SS_DATABASE_SERVER_REPLICA_12` for the 12th replica. Replias must be numbered sequentially starting from `01`.
+
+Replicas cannot define different configuration values for `SS_DATABASE_CLASS`, `SS_DATABASE_NAME`, or `SS_DATABASE_CHOOSE_NAME`. They are restricted to prevent strange issues that could arise from having inconsistent database configurations across replicas.
+
+If one or more read-only replicas have been configured, then for each request one of the read-only replicas will be randomly selected from the pool of available replicas to handle queries for the rest of the request cycle, unless criteria has been met to use the primary database instead, for example a write operation.
+
+See [read-only database replicas](/developer_guides/performance/read_only_database_replicas/) for more details.
+
+Calling the methods [`DB::get_conn()`](api:SilverStripe\ORM\DB::get_conn()), [`DB::set_conn()`](api:SilverStripe\ORM\DB::set_conn()), [`DB::getConfig()`](api:SilverStripe\ORM\DB::getConfig()), and [`DB::setConfig()`](api:SilverStripe\ORM\DB::setConfig()) using the default `$name` parameter of "default" will now emit a deprecation notice and will be internall converted to "primary". Pass [`DB::CONN_PRIMARY`](api:SilverStripe\ORM\DB::CONN_PRIMARY) or simply "primary" for the `$name` parameter instead. The default value for `$name` will be changed to "primary" in a future major release.
```

Comment:
No deprecation notice - looks like these docs may not have been updated to reflect that it's now CMS 6? Don't know how I missed that before.

Reply:
Updated, have changed to mention that "default" has changed to "primary"